examples/Simple Vertically Partitioned SplitNN.ipynb

```diff
@@ -135,26 +127,34 @@
     "\n",
     "from src.dataloader import VerticalDataLoader\n",
     "from src.dataset import add_ids\n",
+    "from src.psi.util import compute_psi\n",
     "\n",
     "hook = sy.TorchHook(torch)"
    ]
   },
   {
    "cell_type": "code",
-   "execution_count": 5,
+   "execution_count": 4,
    "metadata": {},
    "outputs": [],
    "source": [
     "# Create dataset\n",
     "data = add_ids(MNIST)(\".\", download=True, transform=ToTensor())  # add_ids adds unique IDs to data points\n",
     "\n",
     "# Batch data\n",
-    "dataloader = VerticalDataLoader(data, batch_size=128) # partition_dataset uses by default \"remove_data=False, keep_order=True\""
+    "dataloader = VerticalDataLoader(data, batch_size=128) # partition_dataset uses by default \"remove_data=True, keep_order=False\"\n",
+    "\n",
+    "# Compute private set intersections\n",
+    "intersection1 = compute_psi(dataloader.dataloader1.dataset.get_ids(), dataloader.dataloader2.dataset.get_ids())\n",
+    "intersection2 = compute_psi(dataloader.dataloader2.dataset.get_ids(), dataloader.dataloader1.dataset.get_ids())\n",
+    "\n",
+    "# Order data\n",
+    "dataloader.drop_non_intersecting(intersection1, intersection2)"
```

Comment:
Could you print out the first ~10 entries of each dataset (plotting the images) before and after `drop_non_intersecting`, so we can see that the datasets have been aligned